README.md

```diff
@@ -76,6 +76,18 @@ If the export process was successful, you should see three files": "bar.stl", "b
 The YAML format is used to pass parameters to the plugin to customized the conversion process.
 The parameter accepted by the plugin are documented in the following.
 
+
+
+##### Include Parameters
+The YAML can eventually include other YAML files using the `includes` parameter. If so, the included files are merged with the main file, with the included file taking precedence over the main file in case of conflicts.
```

Comment:
It may be worth to mention that also all internal group are merged if they are maps, they are overwritten only if an element is not a map.